### ISSUE TITLE
Give all menu items letter-underlined keyboard shortcuts

### ISSUE THREAD

**Issue by kendonB** [rstudio/rstudio#5610]
This one is a bit of a barrier for users who don't use mice or want to reduce their mouse use.

e.g. "Open Project In New Session", "Modify Keyboard Shortcuts"

It's fine to double them up as well (Windows does this) as users can cycle through them quite quickly if there are just two, for example, "m"s.

**Comment by gtritchie** [comment]
Also, desktop or server? We do it on Desktop on Windows, but not on RStudio Server.

**Comment by gtritchie** [comment]
Oops, didn't mean to click close!

**Comment by kendonB** [comment]
I'm on Windows 10, RStudio Desktop 1.2.1578. Yes most menu items get the shortcut but not all (two examples given above).

**Comment by kendonB** [comment]
This issue also applies to Ubuntu 18.04 RStudio Desktop 1.3.535

**Comment by R-Hannibal** [comment]
Verified Fixed on: Version 1.3.926

Tested on Windows 10 desktop, all menu entries now have letter underlined keyboard shortcuts assigned (official Windows terminology: 'Menu Access Keys').

For certain letters (examples: 'j' and 'i'), the Menu Access Key can be difficult to see. This is a known issue that is native to Windows and cannot be fixed on our end (example included below).
 
![Recent Projects_underline shortcut](https://user-images.githubusercontent.com/62304541/77660046-d464ef80-6f46-11ea-9397-1fd9b4f5d6ff.png)
